src/main/resources/static/css/styles.css

```diff
@@ -1,13 +1,45 @@
 #forecast-screen {
-  margin-top: 10px;
-  margin-left: 10px;
-  margin-right: 10px;
-  margin-bottom: 10px;
+	margin: 10px;
```

Comment:
Margin can be declared using its shorthand way, instead of listing out each size of the margin sides.
**https://www.w3schools.com/css/css_margin.asp**